layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayout.kt

```diff
@@ -94,9 +94,11 @@ open class FoldableLayout @JvmOverloads constructor(
         job = MainScope().launch {
             WindowInfoTracker.getOrCreate(context)
                 .windowLayoutInfo(context as Activity)
-                .collect { info ->
-                    viewModel.windowLayoutInfo = info
-                    layoutController.foldingFeature = info.getFoldingFeature()
+                .collectIndexed { index, info ->
```

Comment:
Don't know but maybe is already an operator that does the whole thing? (collectIndexed + get the first)? something like component1 or similar?

Reply:
I checked and I didn't find something simillar.